build/template-postbuild-code-analysis.yaml

```diff
@@ -6,11 +6,25 @@ steps:
   displayName: 'Run Roslyn Analyzers'
   inputs:
     userProvideBuildInfo: auto
+    suppressionFileForCompilerWarnings: 'microsoft-authentication-library-for-dotnet\RoslynSuppressions.txt'
+    msBuildCommandline: 'dotnet build'
+    rulesetName: Recommended
+    rulesetVersion: Latest
   env:
     system_accesstoken: $(System.AccessToken)
   continueOnError: true
+  condition: and(succeeded(), eq(variables['PipelineType'], 'OneBranch'))
+
+- task: VSBuild@1
+  displayName: 'Rebuild solution for Roslyn workaround'
```

Comment:
What is the workaround for?
Do we really need to build the solution again?
With CodeQL enabled we shouldn't need Roslyn, I think.

Reply:
dead code, will be removed